db/migrate/20170818172013_create_habits.rb

```diff
@@ -1,4 +1,4 @@
-class CreateHabits < ActiveRecord::Migration[5.1]
+habclass CreateHabits < ActiveRecord::Migration[5.1]
```

Comment:
Just have to delete hab here. @sanjaynelson  can you do that on your branch and re-push?